src/frontend/planner_test/tests/testdata/expr.yaml

```diff
@@ -351,3 +351,22 @@
   batch_plan: |
     BatchProject { exprs: ['integer[][]':Varchar] }
     └─BatchValues { rows: [[]] }
+- name: now expression
+  sql: |
+    create table t (v1 timestamp);
+    select * from t where v1 >= now();
+  logical_plan: |
+    LogicalProject { exprs: [t.v1] }
+    └─LogicalFilter { predicate: (t.v1 >= Now) }
+      └─LogicalScan { table: t, columns: [t.v1, t._row_id] }
+  stream_plan: |
+    StreamMaterialize { columns: [v1, t._row_id(hidden)], pk_columns: [t._row_id] }
+    └─StreamDynamicFilter { predicate: (t.v1 >= t.v1), output: [t.v1, t._row_id] }
+      ├─StreamFilter { predicate: true }
```

Comment:
How come the predicate did not get pushed down into the LogicalScan?

Reply:
Hmm, seems like a bug. It does not get pushed down in the usual case too (no `now`)...

Reply:
Here are the planner tests for comparison. Seems like predicates do get pushed down in that case: https://github.com/risingwavelabs/risingwave/blob/main/src/frontend/planner_test/tests/testdata/predicate_pushdown.yaml

Reply:
> How come the predicate did not get pushed down into the LogicalScan?

fixed